internal/cmd/flags.go

```diff
@@ -46,6 +46,8 @@ type flags struct {
 	overwrite      bool
 	pkg            string
 	printFields    string
+	protocBinPath  string
+	protocWktPath  string
```

Comment:
WKT (I think this is what is done elsewhere right?)

Reply:
Yup, good call.